sections/plate_notation.tex

```diff
@@ -87,9 +87,10 @@ \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
 \subsection{Pachinko Allocation}
 In order to handle the hierarchical structure of the taxonomy metadata field, we will be using a hierarchical topic model, namely the \acrlong{pam} from \citet{li2006pachinko}.
 Pachinko allocation generalizes \gls{lda}, to be able to operate on any \gls{dag} structure.
+\vejleder[inline]{struktur over hvad?}
 
 The \gls{dag} structure represents topics which in pachinko allocation are distributions over words and/or other topics further down in the \gls{dag} structure, rather than a distribution only over words as with \gls{lda}.
```

Comment:
```suggestion
The \gls{dag} structure \vejleder[inline]{illustrer gerne} represents topics which in pachinko allocation are distributions over words and/or other topics further down in the \gls{dag} structure, rather than a distribution only over words as with \gls{lda}.
```
Hænger nok sammen med at de snakkede om at DAG træet ikke viser noget spændende lige nu.